userspace/libscap/engine/gvisor/gvisor.cpp

```diff
@@ -41,31 +41,31 @@ namespace scap_gvisor {
 #include <unistd.h>
 #include <utility>
 
-#ifdef __cplusplus
-extern "C"{
-#endif
+extern "C" {
 
-static int32_t scap_gvisor_init_platform(scap_platform* platform, char* lasterr, scap_engine_handle engine, struct scap_open_args* oargs)
+namespace {
```

Comment:
Just curious: why are you suggesting an anonymous namespace rather than C-style static functions? Was there a compiler warning for it? Is there a specific reason? Thanks!

Reply:
They should be equivalent, they give the function internal linkage. I have read on advice to prefer anonymous namespaces for reasons I can't recall, but I suppose it's more a fashion than a deep technical reason.
For this reason in this case I could (should?) have kept the `static`. I probably made it like this because it's more uniform with another PR I have created where the anonymous namespace was created to put inside a class definition that had a duplicate name so that the duplication could be handled by the linker.
Not a problem if you prefer to keep the old syntax, the only advantage here could be a shorter function signature to read.
FTR, the definition of the object `scap_gvisor_engine` has been put inside the `extern "C"` as its declaration elsewhere is inside such a block.